cfgov/v1/atomic_elements/molecules.py

```diff
@@ -12,10 +12,14 @@ def isRequired(field_name):
 
 
 class HalfWidthLinkBlob(blocks.StructBlock):
+    h_text = ('A list of icon names can be obtained at: '
```

Comment:
Minor: I think I like what you're doing here, but I feel like the variable name should be more specific. Perhaps `icon_help_text` or `icon_help`?